tests/integration/cmd_pref_config_test.go

```diff
@@ -306,4 +306,26 @@ var _ = Describe("odo preference and config command tests", func() {
 			os.Setenv("KUBECONFIG", kubeconfigOld)
 		})
 	})
+
+	Context("when using --now with config command", func() {
+		JustBeforeEach(func() {
+			context = helper.CreateNewContext()
+			os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "preference.yaml"))
+		})
+		JustAfterEach(func() {
+			helper.DeleteDir(context)
+			os.Unsetenv("GLOBALODOCONFIG")
+		})
+
+		It("should successfully set and unset variables", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", project, "--context", context)
+			helper.CmdShouldPass("odo", "config", "set", "--now", "--env", "hello=world", "--context", context)
+			configValue1 := helper.CmdShouldPass("odo", "config", "view", "--context", context)
+			helper.MatchAllInOutput(configValue1, []string{"hello", "world"})
```

Comment:
@mohammedzee1000 AFAIU config set using ```--now``` flag directly reflect the config in the dc without ```odo push --config```, isn't it ?. if yes then for cross verification please verify the same config in dc as well.

Reply:
Hmm I guess that could be a work around. Although preferably we want `odo config list` to tell us what we need to know

Reply:
@mohammedzee1000 I have the same question as @amitkrout . Shouldn't this start the push operation after the `set` command is successful? If so please add the verification for the same.

Reply:
@mik-dass @amitkrout done